pyproject.toml

```diff
@@ -6,7 +6,7 @@ authors = [
 description = "A lightweight Python package installer for the web "
 readme = "README.md"
 license = { file="LICENSE" }
-requires-python = ">=3.10"
+requires-python = ">=3.11"
```

Comment:
Not sure about this though. It costs us nothing to keep to support 3.10 for now. Ideally tested in CI but even without testing. So that people could use future releases of micropip with the current release of pyodide.

If we support Pyodide 0.21.3, we need to support the corresponding Python version.

Reply:
> So that people could use future releases of micropip with the current release of pyodide. If we support Pyodide 0.21.3, we need to support the corresponding Python version.

Yes, that makes sense. I'll remove that part.